index.html

```diff
@@ -5,6 +5,16 @@
     <meta charset="UTF-8" />
     <link rel="manifest" href="./manifest.json" />
 
+    <meta property="og:title" content="Databased" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://databased.csa.iisc.ac.in" />
+    <meta property="og:image" content="https://databased.csa.iisc.ac.in/img/banner.webp" />
+    <meta property="og:description" content="Databased is the Undergraduate Computer Science club at IISc. We host competitive programming sessions, Capture the Flag events, Tech talks, alumni workshops, fun, hands-on learning sessions and much more — diving deeper into the world of computer science together.">
```

Comment:
I think its better to have something else like workshops, talks in the front. The single line description would also suffice.